src/Pages/Quiz/Quiz.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Question } from "../../components";
+const Quiz = () => {
+  return (
+    <>
+      <Question />
+    </>
```

Comment:
no need to have fragment, since you only have one component to return, you can return it directly

Reply:
Thanks for pointing out, initially i thought of passing props down but the code was getting hard to read so i decided to make context.
I have refactored it and removed Quiz page